ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.txt

```diff
@@ -179,12 +188,241 @@ hideall "--sync--"
+2038.6 "--targetable--"
+2039.3 "Skyblind 4" sync / 1[56]:[^:]*:Ser Charibert:631A:/
+# Enrage??
 
 
 ### Phase 5: King Thordan II
 3000.0 "Incarnation" sync / 1[56]:[^:]*:King Thordan:6B86:/ window 1000,0
+3006.2 "The Dragon's Eye" sync / 1[56]:[^:]*:King Thordan:6B87:/
+3016.7 "--sync--" sync / 1[56]:[^:]*:King Thordan:63C4:/
+3017.8 "Knights of the Round" sync / 1[56]:[^:]*:King Thordan:63ED:/
+3024.9 "Wrath of the Heavens" sync / 1[56]:[^:]*:King Thordan:6B89:/
+3034.4 "Chain Lightning" sync / 1[56]:[^:]*:Darkscale:6B8F:/
+3036.8 "Twisting Dive" sync / 1[56]:[^:]*:Vedrfolnir:6B8B:/
+3036.8 "Skyward Leap" sync / 1[56]:[^:]*:Ser Paulecrain:72A2:/
+3036.8 "Spiral Pierce" sync / 1[56]:[^:]*:Ser Ignasse:6B8A:/
+3042.2 "Ascalon's Mercy Revealed" sync / 1[56]:[^:]*:King Thordan:63CA:/
+3043.1 "Liquid Heaven 1" #sync / 1[56]:[^:]*:Vedrfolnir:6B91:/
+3044.3 "Liquid Heaven 2" #sync / 1[56]:[^:]*:Vedrfolnir:6B91:/
+3045.5 "Liquid Heaven 3" #sync / 1[56]:[^:]*:Vedrfolnir:6B91:/
+3046.6 "--sync--" sync / 1[56]:[^:]*:Ser Charibert:63E4:/
+3046.7 "Liquid Heaven 4" #sync / 1[56]:[^:]*:Vedrfolnir:6B91:/
+3047.1 "Altar Flare 1" #sync / 1[56]:[^:]*:Ser Charibert:63E5:/
+3047.9 "Liquid Heaven 5" #sync / 1[56]:[^:]*:Vedrfolnir:6B91:/
+3048.7 "Altar Flare 2" #sync / 1[56]:[^:]*:Ser Charibert:63E5:/
+3050.1 "Cauterize" #sync / 1[56]:[^:]*:Darkscale:6B8D:/
+3050.2 "Altar Flare 3" #sync / 1[56]:[^:]*:Ser Charibert:63E5:/
+3050.3 "Empty Dimension" sync / 1[56]:[^:]*:Ser Grinnaux:62DA:/
+3051.7 "Altar Flare 4" #sync / 1[56]:[^:]*:Ser Charibert:63E5:/
+3057.4 "Ancient Quaga" sync / 1[56]:[^:]*:King Thordan:63C6:/
+3067.8 "Heavenly Heel" sync / 1[56]:[^:]*:King Thordan:63C7:/
+3071.0 "Ascalon's Might 1" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+3072.6 "Ascalon's Might 2" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+3074.2 "Ascalon's Might 3" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+3077.0 "--sync--" sync / 1[56]:[^:]*:King Thordan:63C4:/
+3078.1 "Knights of the Round" sync / 1[56]:[^:]*:King Thordan:63ED:/
+3085.2 "Death of the Heavens" sync / 1[56]:[^:]*:King Thordan:6B92:/
+3092.2 "Deathstorm" sync / 1[56]:[^:]*:Darkscale:6B94:/
+3100.4 "Heavy Impact 1" sync / 1[56]:[^:]*:Ser Guerrique:63D6:/
+3102.0 "Lightning Storm" sync / 1[56]:[^:]*:King Thordan:63CC:/
+3102.3 "Twisting Dive" #sync / 1[56]:[^:]*:Vedrfolnir:6B8B:/
+3102.3 "Cauterize" #sync / 1[56]:[^:]*:Darkscale:6B8D:/
+3102.3 "Spear of the Fury" #sync / 1[56]:[^:]*:Ser Zephirin:6B93:/
+3102.3 "Heavy Impact 2" #sync / 1[56]:[^:]*:Ser Guerrique:63D7:/
+3102.4 "Lightning Storm" #sync / 1[56]:[^:]*:King Thordan:63CD:/
+3104.1 "Heavy Impact 3" sync / 1[56]:[^:]*:Ser Guerrique:63D8:/
+3106.0 "Heavy Impact 4" sync / 1[56]:[^:]*:Ser Guerrique:63D9:/
+3107.9 "Heavy Impact 5" sync / 1[56]:[^:]*:Ser Guerrique:63DA:/
+3114.4 "The Dragon's Gaze" sync / 1[56]:[^:]*:King Thordan:63D0:/
+3115.5 "Faith Unmoving" #sync / 1[56]:[^:]*:Ser Grinnaux:62DC:/
+3115.5 "The Dragon's Gaze" #sync / 1[56]:[^:]*:King Thordan:63D1:/
```

Comment:
This is the damage from the lookaway, I think.

Reply:
I have removed it. I see 63D0 is also ignored in the previous King Thordan phase.